wp-content/themes/redmond/views/layout/footer.html.twig

```diff
@@ -1,3 +1,37 @@
 <footer class="footer">
-
-</footer>
\ No newline at end of file
+    <div class="row top-row">
+        {% for column in SP.ACF.get_field('footer_icons', 'option') %}
+            <div class="large-4 medium-4 columns icon-title">
+                <a href="#"><img class="footer-icon" src="{{ column.icon.url }}" /></a>
+                <h2><small>{{ column.small_header }}</small> {{ column.large_header }}</h2>
+            </div>
+        {% endfor %}
+    </div>
+    <div class="row">
+        {% for column in SP.ACF.get_field('footer_list', 'option') %}
+            <div class="column {% if loop.index == 1 %}hide-for-small-only medium-3 medium-push-3 large-3 large-push-3{% endif %}{% if loop.index == 2 %}hide-for-small-only medium-3 medium-push-3 large-3 large-push-3{% endif %}{% if loop.index == 3 %}small-6 medium-3 medium-push-3 large-3 large-push-3{% endif %}">
+                <h2>{{ column.title }}</h2>
+            </div>
+        {% endfor %}
+        <div class="column large-3 medium-3 small-6 medium-pull-9 large-pull-9 centered">
+            <img class="logo" src="{{ SP.ACF.get_field('logo', 'option').url }}" alt="" />
+            <div class="row">
+                {% if SP.ACF.get_field('facebook', 'option') %}
+                    <a href="{{ SP.ACF.get_field('facebook') }}"><div class="facebook"><i class="fa fa-facebook" aria-hidden="true"></i></div></a>
+                {% endif %}
+                {% if SP.ACF.get_field('twitter', 'option') %}
+                    <a href="{{ SP.ACF.get_field('twitter') }}"><div class="twitter"><i class="fa fa-twitter" aria-hidden="true"></i></div></a>
+                {% endif %}
+                {% if SP.ACF.get_field('pinterest', 'option') %}
+                    <a href="{{ SP.ACF.get_field('pinterest') }}"><div class="pinterest"><i class="fa fa-pinterest-p" aria-hidden="true"></i></div></a>
+                {% endif %}
+                {% if SP.ACF.get_field('google_plus', 'option') %}
+                    <a href="{{ SP.ACF.get_field('google_plus') }}"><div class="google_plus"><i class="fa fa-google-plus" aria-hidden="true"></i></div></a>
+                {% endif %}
+            </div>
+        </div>
+    </div>
+    <div class="row bottom-row">
+        <p>{{ SP.ACF.get_field('copyright', 'option') }}</p>
```

Comment:
remove the p tag. the wysiwyg editor automatically creates the p tags for the user content.